src/main/java/org/testobject/rest/api/resource/v2/TestSuiteResourceV2.java

```diff
@@ -0,0 +1,64 @@
+package org.testobject.rest.api.resource.v2;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import java.util.List;
+
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+@Path("v2/batches")
+public interface TestSuiteResourceV2 {
+
+	@PUT
+	@Path("instrumentation/{testSuite}")
+	public void updateInstrumentationTestSuite(@PathParam("testSuite") long testSuite,
+			InstrumentationTestSuiteRequest request, String apikey);
+
+	@POST
+	@Path("instrumentation/{testSuite}/replay")
+	public long runInstrumentationTestSuite(@PathParam("testSuite") long testSuite, String apikey);
+
+	class InstrumentationTestSuiteRequest {
+
+		@JsonProperty
+		public String appUploadId;
+
+		@JsonProperty
+		public String testUploadId;
+
+		@JsonProperty
+		public List<String> methodsToRun;
+
+		@JsonProperty
+		public List<String> classesToRun;
+
+		@JsonProperty
+		public List<String> annotationsToRun;
+
+		@JsonProperty
+		public List<String> sizesToRun;
+
+		@JsonProperty
+		public String tunnelIdentifier;
+
+		@JsonProperty
+		public boolean runAsPackage;
+
+		@SuppressWarnings("unused")
+		private InstrumentationTestSuiteRequest() {
```

Comment:
why do we need a private constructor with unused suppresswarnings?

Reply:
no idea. I just modified paths and rest of class left as it was. There are probably a lot of things what can be removed or cleaned in this java-api project, but I can't take it because not sure 100% where it is used.